frontend/src/model/GameMode.ts

```diff
@@ -0,0 +1,12 @@
+import Settings from './Settings';
+
+class GameMode extends Settings {
+  gameMode: string;
+
+  constructor(turnLength: number, tileAmount: number, boardSize: number, gameMode: string) {
+    super(turnLength, tileAmount, boardSize);
+    this.gameMode = gameMode;
```

Comment:
add specific type type to this gameMode property. How about (Classic, Fast, Custom)?